### DISCUSSION TITLE
Idea: in hassio integration, when PowerOnState is 4, dont have a switch entity

### DISCUSSION THREAD

**Discussion by MarkAtwood** [arendst/Tasmota#16927] [Ideas]
PowerOnState 4 is "turn power(s) ON and disable further power control"

When that is set, it makes since to not announce a switch entity.

**Comment by sfromis** [comment]
You could avoid the unwanted switch entity popping up by instead configuring the pin as `Output Hi` (or `Output Lo` for inverted).
